libs/unity/library/Runtime/Scripts/NativeRender/NativeVideoRenderer.cs

```diff
@@ -0,0 +1,219 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using UnityEngine;
+using Microsoft.MixedReality.WebRTC.UnityPlugin;
+using UnityEngine.UI;
+
+namespace Microsoft.MixedReality.WebRTC
```

Comment:
```suggestion
namespace Microsoft.MixedReality.WebRTC.Unity
```
For consistency with the other components in the package.

Reply:
Updated